ios/test_app.rb

```diff
@@ -53,7 +53,12 @@ def nearest_node_modules(project_root)
 
 def package_version(package_path)
   package_json = JSON.parse(File.read(File.join(package_path, 'package.json')))
-  package_json['version'].match(/(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/)
+  match = package_json['version'].match(/(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/)
+
+  # Convert #<MatchData "0.62.2" major:"0" minor:"62" patch:"2">
+  #      => {:major=>0, :minor=>62, :patch=>2}
+  # Source: https://gist.github.com/bchase/d9ecfc1d58e38f158793525864583b17
+  [match.names.map(&:to_sym), match.captures.map(&:to_i)].transpose.to_h
```

Comment:
The Ruby stdlib should include RubyGems iirc, which means you may just as well require its `Version` class: https://github.com/rubygems/rubygems/blob/master/lib/rubygems/version.rb

Reply:
Is that something that needs to be installed separately? I don't see it in `gem list`.

Reply:
No, because stdlib libraries won’t show up in `gem list`:

```
irb(main):001:0> require "rubygems/version"
=> false
irb(main):002:0> Gem::Version.new("0.63.0-rc.1") < Gem::Version.new("0.63.0-rc.2")
=> true
irb(main):003:0> Gem::Version.new("0.63.0-rc.1") < Gem::Version.new("0.63.0-rc.0")
=> false
```

Reply:
Thanks. This looks nicer.